libs/application/ui-shell/src/components/FormExternalDataProvider.tsx

```diff
@@ -162,13 +170,15 @@ const FormExternalDataProvider: FC<{
                     activateBeforeSubmitCallback(isChecked)
                   }}
                   checked={value}
+                  hasError={error !== undefined}
```

Comment:
Makes sense, I will move it :)

Reply:
Actually tested it and the rendering is way off if I put it in the checkbox. 
Don't want to change the checkbox because it makes sense the way it is currently if there is no `Box` around it.